src/app/pages/institutional-partnership/institutional-partnership.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import LoaderPage from '~/components/jsx-helpers/loader-page';
+import LazyLoad from 'react-lazyload';
+import {camelCaseKeys} from '~/helpers/page-data-utils';
+import Banner, {BannerProps} from './sections/banner/banner';
+import OverlappingQuote, {OverlappingQuoteProps} from './sections/overlapping-quote/overlapping-quote';
+import About, {AboutProps} from './sections/about/about';
+import Promoting, {PromotingProps} from './sections/promoting/promoting';
+import BigQuote, {BigQuoteProps} from './sections/big-quote/big-quote';
+import Speaking, { SpeakingProps } from './sections/speaking/speaking';
+import Results, { ResultsProps } from './sections/results/results';
+import Participants, { ParticipantsProps } from './sections/participants/participants';
+import SmallQuote, { SmallQuoteProps } from './sections/small-quote/small-quote';
+import SignUp, { SignUpProps } from './sections/sign-up/sign-up';
+import './institutional-partnership.scss';
+
+type DataObject = Record<string, unknown>;
+
+type ImageMeta = {
+    download_url: string;
+};
+
+type SectionData = {
+    section_4_background_image: {
+        meta: ImageMeta;
+    };
+} & DataObject;
+
+function unprefixKey(newObject: DataObject, oldKey: string, prefix: string, data: DataObject) {
+    const newKey = oldKey.replace(prefix, '');
+
+    newObject[newKey] = data[oldKey];
+    return newObject;
+}
+
+function sectionData(data: DataObject, sectionNumber: string | number) {
+    const sectionPrefix = `section_${sectionNumber}_`;
+
+    return camelCaseKeys(
+        Reflect.ownKeys(data)
+            .filter((k) => String(k).startsWith(sectionPrefix))
+            .reduce(
+                (a, oldKey) => unprefixKey(a, String(oldKey), sectionPrefix, data), {}
+            )
```

Comment:
Do we have other CMS data that needs this transformation? I can see this being a reusable helperDo we have other CMS data that needs this transformation? I can see this being a reusable helper